Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.4
+// swift-tools-version:5.7
```

Comment:
From now on, SwiftLayout requires swift 5.7.

We need to manage the swift version.

Reply:
@oozoofrog @della-padula 
I will put a table in README.md that summarizes the versions of Swift that can be used for each version.
In the case of the latest version, only 5.7 will be supported.